pypose/function/geometry.py

```diff
@@ -159,3 +161,134 @@ def reprojerr(points, pixels, intrinsics, extrinsics=None):
            intrinsics.size(-1) == intrinsics.size(-2) == 3, "Shape not compatible."
     img_repj = point2pixel(points, intrinsics, extrinsics)
     return (img_repj - pixels).norm(dim=-1)
+
+
+def knn(pc1, pc2, ord=2, k=1, dim=-1, largest=False, sorted=True):
```

Comment:
![image](https://user-images.githubusercontent.com/48173064/234433957-84fea023-d958-4ad3-b69f-9ac58acdc0a7.png)
Would anyone be able to help in resolving the annoying spacing issue present in `dim=-1`?